.asf.yaml

```diff
@@ -16,7 +16,7 @@
 #
 
 github:
-  description: Apache DolphinScheduler is the modern data orchestration platform. Agile to create high performance workflow with low-code
```

Comment:
Seems not related to this PR?

Reply:
Yes, I didn't make any special changes to this. It's possible that I mistook it when synchronizing the latest code